### ISSUE TITLE
PHP Notice when Jetpack deactivated: Use of undefined constant JETPACK__VERSION

### ISSUE THREAD

**Issue by allendav** [Automattic/woocommerce-services#1249]
Observed with Jetpack 5.5.1 and WooCommerce Services 1.9.0 and WP_DEBUG true

```

( ! ) Notice: Use of undefined constant JETPACK__VERSION - assumed 'JETPACK__VERSION' in /home/allendav/public_html/wp-content/plugins/woocommerce-services/woocommerce-services.php on line 1042
--


1 | 0.0020 | 479688 | {main}( ) | .../plugins.php:0
2 | 0.0028 | 526808 | require_once( '/home/allendav/public_html/wp-admin/admin.php' ) | .../plugins.php:10
3 | 0.8954 | 44719880 | do_action( ) | .../admin.php:156
4 | 0.8954 | 44720256 | WP_Hook->do_action( ) | .../plugin.php:453
5 | 0.8954 | 44720256 | WP_Hook->apply_filters( ) | .../class-wp-hook.php:310
6 | 0.9058 | 44950144 | WC_Connect_Loader->admin_enqueue_scripts( ) | .../class-wp-hook.php:286
```

<img width="1430" alt="screen shot 2017-11-28 at 8 23 12 am" src="https://user-images.githubusercontent.com/1595739/33331128-9ccf2a6e-d415-11e7-9570-871d3f0e9d05.png">

**Comment by jeffstieler** [comment]
Are you on the latest `master`?

https://github.com/Automattic/woocommerce-services/issues/1241

**Comment by allendav** [comment]
No - released 1.9.0

**Comment by jeffstieler** [comment]
Closed in https://github.com/Automattic/woocommerce-services/issues/1241.
